week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -14,11 +14,11 @@
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  3. What is printed to the console? (2 6 4 9 6 13 8 )
+  4. How many times is "f1" called? 2
+  5. How many times is "f2" called? 5
```

Comment:
Could you please explain how you got this count?

Reply:
I used break points :)

Reply:
It's a good tactic :) but you must have miscounted at some point.
If there is only 5 iteration and we call `f2` only when `if (i % 2 === 0)` is true than `f2` cannot be invoked 5 times.